patterns/llm.py

```diff
@@ -0,0 +1,81 @@
+from langchain.chat_models import ChatOpenAI
+from langchain.llms import OpenAI
+
+from resolver import pattern
+
+from .secret import Secret
+
+
+@pattern(name="OpenAI_Complete_LLM")
+class OpneAIWrapper(OpenAI):
+    """
+    A wrapper class for interacting with the OpenAI API for complete language models.
+    """
+
+    def __init__(
+        self,
+        openai_api_key: Secret,
+        temperature: float = 0,
+        max_tokens: int = 2048,
+        model_name: str = "text-davinci-003",
```

Comment:
This model is deprecated. See [here](https://platform.openai.com/docs/deprecations/instructgpt-models).

Reply:
I'll adjust the defalut value to `gpt-3.5-turbo-instruct`